ggml-cuda.cu

```diff
@@ -180,6 +180,12 @@ static_assert(sizeof(half) == sizeof(ggml_fp16_t), "wrong fp16 size");
     } while (0)
 #endif // CUDART_VERSION >= 11
 
+#if CUDART_VERSION >= 11100
+#define GGML_ASSUME(x) __builtin_assume(x)
+#else
+#define GGML_ASSUME(x)
+#endif // CUDART_VERSION >= 11100
```

Comment:
I few weeks ago I also made a PR for this but no one ever reviewed it: https://github.com/ggerganov/llama.cpp/pull/2669 . The solution was different but something like this would also be fine. However, please rename the macro from `GGML_ASSUME` to `GGML_CUDA_ASSUME` to avoid confusion regarding where it is being defined.

Reply:
Thanks for pointing out, fixed it. Let's hope that at least one of our PRs gets merged:)